### ISSUE TITLE
Internal Exception: io.netty.handler.codec.DecoderException...

### ISSUE THREAD

**Issue by L4BORG** [MylesIsCool/ViaVersion#35]
```
Internal Exception: io.netty.handler.codec.DecoderException: com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: User JsonReader.setLenient(true) to accept malformed JSON at line 1 column 2
```

Any idea why I would always just get disconnected or get one of those exceptions with 1.9 client on latest BungeeCord and latest Spigot 1.8.9?

**Comment by MylesIsCool** [comment]
What plugins are you using?

**Comment by L4BORG** [comment]
```
>pl
[15:49:37 INFO]: Plugins (20): AsyncWorldEditInjector, NoCheatPlus, AutoSaveWorld, WorldEdit, AsyncWorldEdit, OpenInv, LogBlock, Buycraft, ViaVersion, WorldBorder, HolographicDisplays, Vault, WorldGuard, BungeePortals, Essentials, EchoPet, TrailGUI
```

Plugin is doing this? HolographicDisplays maybe?

**Comment by ajthemacboy** [comment]
I had this error until I used the latest version compiled from source.

**Comment by MylesIsCool** [comment]
Fixed in 0.4.0
